packages/core/src/App/Containers/Layout/header/dtrader-header.tsx

```diff
@@ -0,0 +1,140 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { routes, getDecimalPlaces, platforms } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+// import { useWalletMigration } from '@deriv/hooks';
+import { MenuLinks, PlatformSwitcher } from 'App/Components/Layout/Header';
+import platform_config from 'App/Constants/platform-config';
+import RealAccountSignup from 'App/Containers/RealAccountSignup';
+import SetAccountCurrencyModal from 'App/Containers/SetAccountCurrencyModal';
+import NewVersionNotification from 'App/Containers/new-version-notification.jsx';
+import ToggleMenuDrawer from 'App/Components/Layout/Header/toggle-menu-drawer.jsx';
+import { AccountsInfoLoader } from 'App/Components/Layout/Header/Components/Preloader';
+import TradersHubHomeButton from './traders-hub-home-button';
+import HeaderAccountActions from './header-account-actions';
+
+const DTraderHeader = observer(() => {
+    const { client, common, ui, notifications, traders_hub } = useStore();
+    const {
+        currency,
+        has_any_real_account,
+        is_bot_allowed,
+        is_dxtrade_allowed,
+        is_logged_in,
+        is_logging_in,
+        is_mt5_allowed,
+        is_virtual,
+        is_switching,
+    } = client;
+    const { app_routing_history, platform, current_language } = common;
+    const { header_extension, is_app_disabled, is_desktop, is_mobile, is_route_modal_on, toggleReadyToDepositModal } =
+        ui;
+    const { addNotificationMessage, client_notifications, removeNotificationMessage } = notifications;
+    const { setTogglePlatformType } = traders_hub;
+
+    const history = useHistory();
+
+    const addUpdateNotification = () => addNotificationMessage(client_notifications?.new_version_available);
+    const removeUpdateNotification = React.useCallback(
+        () => removeNotificationMessage({ key: 'new_version_available' }),
+        [removeNotificationMessage]
+    );
```

Comment:
I see this is being used in both `dtrader-header` and `default header` should we move it to a constant file?

Reply:
client_notifications and removeNotificationMessage are taken from store. so i think moving this to constant file is not practical. or am i missing out on something?